bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bootstrap-social",
-  "main": "bootstrap-social.css",
+  "main": ["bootstrap-social.css", "_bootstrap-social.scss"],
```

Comment:
there is no underscore.. and we should then the *.less as well.. if so. split the array in multiple lines..